karma.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
I think I know how to get coverage with isparta instrumenter, babel and istanbul. Will make another PR for that (hopefully today).